smart-contracts-hardhat/scripts/deployUniversalProfiileWithLSP23.ts

```diff
@@ -0,0 +1,149 @@
+// libs
+import { ethers } from 'hardhat';
+import { AbiCoder } from 'ethers';
+import { ERC725 } from '@erc725/erc725.js';
+
+// LSPs artifacts
+import LSP23FactoryArtifact from '@lukso/lsp-smart-contracts/artifacts/LSP23LinkedContractsFactory.json';
+import UniversalProfileInitArtifact from '@lukso/lsp-smart-contracts/artifacts/UniversalProfileInit.json';
+
+// ERC725.js schemas
+import LSP1UniversalReceiverDelegateSchemas from '@erc725/erc725.js/schemas/LSP1UniversalReceiverDelegate.json';
+import LSP3ProfileMetadataSchemas from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json';
+import LSP6KeyManagerSchemas from '@erc725/erc725.js/schemas/LSP6KeyManager.json';
+
+const LSP23_FACTORY_ADDRESS = '0x2300000A84D25dF63081feAa37ba6b62C4c89a30';
+const LSP23_POST_DEPLOYMENT_MODULE = '0x000000000066093407b6704B89793beFfD0D8F00';
+const UNIVERSAL_PROFILE_IMPLEMENTATION_ADDRESS = '0x3024D38EA2434BA6635003Dc1BDC0daB5882ED4F';
+const LSP6_KEY_MANAGER_IMPLEMENTATION_ADDRESS = '0x2Fe3AeD98684E7351aD2D408A43cE09a738BF8a4';
+const UNIVERSAL_RECEIVER_ADDRESS = '0x7870C5B8BC9572A8001C3f96f7ff59961B23500D'; // this will be needed later so we can set the Universal Receiver to the Universal Profile (see https://docs.lukso.tech/standards/generic-standards/lsp1-universal-receiver)
```

Comment:
Suggest renaming + putting commment at the top to avoid right scrolling
And writing it as a JS Docs for more emphasis on docs link too :)

```suggestion
/**
 * This will be needed later so we can set the Universal Receiver to the Universal Profile 
 * @link See https://docs.lukso.tech/standards/generic-standards/lsp1-universal-receiver
 */
const LSP1_UNIVERSAL_RECEIVER_ADDRESS = '0x7870C5B8BC9572A8001C3f96f7ff59961B23500D'; 
```